adabot_description/urdf/adabot.macros.xacro

```diff
@@ -2,30 +2,6 @@
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="adabot">
 
 
-    <!-- Create the chassis geometry (for visuals and collisions) -->
-    <xacro:property name="chassis_geometry">
-        <origin xyz="0 0 0" rpy="0 0 0" />
-        <geometry>
-            <box size="${ch_length} ${ch_width} ${ch_height}" />
-        </geometry>
-    </xacro:property>
-
-
-
-
-    <!-- Create the wheel geometry (for visuals and collisions) -->
-    <xacro:property name="wheel_geometry">
-        <!-- Cylinders default with their height along the z axis -->
-        <origin xyz="0 0 0" rpy="${PI/2} 0 0" />
-        <geometry>
-            <!-- <cylinder length="${wh_length}" radius="${wh_radius}" /> -->
```

Comment:
Wegs can be cylinders (`debug_visuals:=true`) or STL meshes (`debug_visuals:=false`).